hash.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Security;
+using System.Security.Cryptography;
```

Comment:
[Do not use broken/weak cryptographic hash algorithms](#CROSS001)
Sensitive namespace is being used